http/src/main/java/io/micronaut/http/hateoas/DefaultLink.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2017-2018 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.micronaut.http.hateoas;
+
+import io.micronaut.http.MediaType;
+
+import java.net.URI;
+import java.util.Optional;
+
+/**
+ * Default implementation of {@link Link}.
+ *
+ * @author Graeme Rocher
+ * @since 1.1
```

Comment:
I've added the `@since 1.1` (forecasting that this will enter the 1.1.x release)

Reply:
Looks good